flink-runtime-web/web-dashboard/src/app/services/artifact.service.ts

```diff
@@ -26,12 +26,12 @@ import { catchError, map } from 'rxjs/operators';
 @Injectable({
   providedIn: 'root'
 })
-export class JarService {
+export class ArtifactService {
   /**
-   * Get uploaded jar list
+   * Get uploaded artifact list
    */
-  loadJarList() {
-    return this.httpClient.get<JarListInterface>(`${BASE_URL}/jars`).pipe(
+  loadArtifactList() {
+    return this.httpClient.get<JarListInterface>(`${BASE_URL}/artifacts`).pipe(
```

Comment:
Should we rename the `JarListInterface` to `ArtifactListInterface` in `jar.js`?

Reply:
I think so. +1